plugin-manifest.js

```diff
@@ -1,2 +1,2 @@
-const plugins = [];
+const plugins = [{ id: 'com.robzienert.helloworld', version: 'snapshot', url: '/plugins/helloworld/index.js' }];
```

Comment:
Will adding this is without the corresponding `index.js` file cause any issues?

Reply:
oh wow, nice catch!  That's not supposed to be there.